js/src/rive.ts

```diff
@@ -0,0 +1,965 @@
+import Runtime from '../../wasm/publish/rive.js';
+
+// #region LoopEvent
+
+// Loop types. The index of the type is the value that comes from Wasm
+export const loopTypes: string[] = ['oneShot', 'loop', 'pingPong'];
```

Comment:
Small typescript trick, as this array should read only, you can do 
```typescript
export const loopTypes = ['oneShot', 'loop', 'pingPong'] as const;
export type LoopType = typeof loopTypes[number];
```
And you derive a string literal type out of your array. Which reduce the potential typo errors

Reply:
Good catch; I'm going to move this to a string enum, which will give us consistency with the other string array types.